requests/__init__.py

```diff
@@ -83,8 +90,16 @@ def check_compatibility(urllib3_version, chardet_version):
 try:
     from urllib3.contrib import pyopenssl
     pyopenssl.inject_into_urllib3()
+
+    # Check pyOpenSSL version
+    cryptography_version = sys.modules['cryptography'].__version__
```

Comment:
No need to pass both of these. We don't actually care about the pyOpenSSL version as I stated above, just that it's being imported for urllib3.

We can do the check with something like:
```
from cryptography import __version__ as cryptography_version
```

This allows us to have to only worry about the import error.